src/components/button/button.tsx

```diff
@@ -3,7 +3,8 @@ import { StyledButton } from "./styled"
 import { Icon } from "../icon"
 import { LoaderIcon } from "../icon/components"
 
-export type ButtonType = "default" | "hollow" | "borderless"
+export type ButtonFlavour = "default" | "hollow" | "borderless"
+export type ThemeType = "light" | "dark"
```

Comment:
should Theme be imported from other, common place?

Reply:
Of course! This is just for the cases where we have a button on a dark background, and we need to override the button theme in place. In general, the only way to change the theme is through the main themeProvider mechanism.

The main example for overriding needs is when we have a simple borderless button on a black background, where we need it to work always like it's in "dark" mode. Otherwise, grey text won't look good on black background :)

Reply:
that's good to know, thx!